### ISSUE TITLE
checkra1n unable to fetch bootstraps

### ISSUE THREAD

**Issue by githubuser1323** [checkra1n/BugTracker#1772]
pls help me someone i have reinstalled checkra1n multiple times and have recieved "unable to fetch bootstraps" everytime i open it, i have checked the date and time and wifi and they are correct and there is neither no icloud or apple id connected pls help as i dont know any other options to fix this?

**Comment by Siguza** [comment]
Can you access this URL on your device?: https://assets.checkra.in/loader/config.json

**Comment by DeerSpotter** [comment]
> Can you access this URL on your device?: https://assets.checkra.in/loader/config.json

Just for the fun of it I tried: https://assets.checkra.in

Had a little Chuckle

**Comment by solarkraft** [comment]
> Had a little Chuckle

Hehe. 

> Can you access this URL on your device?: https://assets.checkra.in/loader/config.json

Opening it once solved the issue on my SE with 14.4.

**Comment by TSP-wengle** [comment]
hi, i can't access https://assets.checkra.in/loader/config.json, website is down?

**Comment by Siguza** [comment]
@TSP-wengle yes it is. Some issue with the hoster, we're trying to get it back up.

**Comment by codewarriormpl** [comment]
i am facing same issue, it is not loading so i am unable to install cydia after i root via checkra

**Comment by simgaymer** [comment]
ditto, unable to fetch bootstraps at this time. do we have an eta for when the site will be back up?

**Comment by githubuser1323** [comment]
Stop emailing me idc

**Comment by yakuv0** [comment]
checkra1n no internet no problem

https://github.com/yakuv0/locra1n
